lib/aerospike/cluster.rb

```diff
@@ -63,6 +58,13 @@ def initialize(policy, hosts)
       end
 
       initialize_tls_host_names(hosts) if tls_enabled?
+
+      @min_connections_per_node = policy.min_connections_per_node
```

Comment:
This information is already in the `client_policy`. No need to define new class variables to hold it.